src/CAN/FakeCANBoard.cpp

```diff
@@ -33,6 +35,7 @@ enum class TestMode {
 std::unordered_set<int> modes = {
 	static_cast<int>(TestMode::ModeSet),	   static_cast<int>(TestMode::PWM),
 	static_cast<int>(TestMode::PID),		   static_cast<int>(TestMode::Encoder),
+	static_cast<int>(TestMode::PIDVel),		   static_cast<int>(TestMode::Encoder),
```

Comment:
looks like we're repeating `TestMode::Encoder` twice, can you remove the second occurrence?